api-specifications/resources-ds-3.3/README.md

```diff
@@ -0,0 +1,20 @@
+# Ed-Fi Resource API (3.3)
+
+The Ed-Fi Resources API enables applications to read and write education data
+stored in an Ed-Fi-compatible application through a secure REST interface.
+
+These file represent the core Resource API generated for [Ed-Fi Data Standard
+3.3](https://techdocs.ed-fi.org/display/EFDS33/Ed-Fi+Data+Standard+v3.3)
+
+* [OpenAPI specification](resources-ds-3.3.yml)
+
+No Postman file has been saved for this specification. The exported file is
+close to 20 MB, which is excessive for a Git repository. Furthermore, we do not
+have any test automation for that as of yet; the examples in the OpenAPI spec
+are insufficient for auto-generating tests. Since there are not tests, there is
+no advantage to sharing a Postman file here. Anyone with Postman can import the
+OpenAPI specification file directly on their machine.
```

Comment:
@ShawnPetersonImani please see not above. I tried many things to get a smaller file, didn't succeed. Finally realized that we don't really need it right now. Was just a convenience.